pkg/reconciler/kubernetes/tektonpipeline/reconcile.go

```diff
@@ -98,7 +98,8 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tp *v1alpha1.TektonPipel
 	// Mark PreReconcile Complete
 	tp.Status.MarkPreReconcilerComplete()
 
-	if err := r.installerSetClient.MainSet(ctx, tp, &r.manifest, filterAndTransform(r.extension)); err != nil {
+	manifest := r.manifest.Filter(mf.Not(mf.ByKind("Namespace")))
```

Comment:
So when TektonPipeline component was deleted `tekton-pipelines` namespace was getting deleted
because in pipeline reconciler namespace was updated by adding few labels which in turn also updated the ownerRef of `tekton-pipelines` namespace from TektonConfig to TektonPipeline

Since namespace is created in TektonConfig reconciler hence deleting TektonPipeline component should not delete the namespace hence omitting the namespace here

Reply:
@PuneetPunamiya can you please add this notes on the change. So it will be easier to understand in the future.

Reply:
Yeah I have added a comment, will update the patch